TESTS/bug_1315_single.c

```diff
@@ -72,7 +72,7 @@ int main() {
   int i;
   for (i = 0; i < nev; ++i) {
     printf("%f\n", dr[i]);
-    if (fabs(dr[i] - (float)(1000 - i)) > 1e-2) {
+    if (fabs(dr[i] - (float)(1000 - i)) > 1e-1) { // Easy tolerance as we deal with float.
```

Comment:
I don't have a W10 environment to test myself. Since this CI disables MPI, it is true the MPI bug is unlikely to kick in here. 

On Linux I see the deviation being roughly 0.003, not sure about how it jumps to > 0.01 on Windows, but seems possible with float.